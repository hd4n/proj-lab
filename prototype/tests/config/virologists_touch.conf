@fields
em e1
em e2

@neighbors
e1 e2

@virologists
v1 e1 null null null
v2 e2 null null null