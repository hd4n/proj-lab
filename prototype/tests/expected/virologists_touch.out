Load: sikeres

stats v1:
fi_e1
eq_null
ma_null
ef_null
co_null

stats v2:
fi_e2
eq_null
ma_null
ef_null
co_null

stats map:
e1 - e2
e2 - e1